### PR TITLE
fix: copypasta errors for `glab issue view --help`

### LINKED ISSUE #972: `glab issue view --help` has copy-paste errors from `glab mr view --help`
the problem is pretty obvious and even though I have no `go` experience the fix looks trivial so I'm skipping the issue template and will open an MR in a few mins (if for some reason there is any problem with MR I will edit this issue to match the template)
```
❯ glab issue view --help | grep mr
  -c, --comments       Show mr comments and activities
  -w, --web            Open mr in a browser. Uses default browser or browser specified in BROWSER variable
❯ glab mr view --help | grep -E '(comments|web)'
  -c, --comments       Show mr comments and activities
  -w, --web            Open mr in a browser. Uses default browser or browser specified in BROWSER variable
```

### PR DESCRIPTION
**Pull request by justin-f-perez**
Fix `glab issue view --help` descriptions in `FLAGS` section for `--comments` and `--web` (mr -> issue)

## Description
Fix `glab issue view --help` descriptions in `FLAGS` section for `--comments` and `--web` (mr -> issue)

## Related Issue

Resolves #972

## How Has This Been Tested?

**test command**
```bash
❯ git grep -E 'issue.*mr'
```
**before**
```diff
commands/issue/create/issue_create.go:73:                       $ glab issue new -t "Fix CVE-YYYY-XXXX" -l security --linked-mr 123
commands/issue/create/issue_create.go:132:      issueCreateCmd.Flags().IntVarP(&opts.LinkedMR, "linked-mr", "", 0, "The IID of a merge request in which to resolve all issues")
commands/issue/view/issue_view.go:107:  issueViewCmd.Flags().BoolVarP(&opts.ShowComments, "comments", "c", false, "Show mr comments and activities")
commands/issue/view/issue_view.go:109:  issueViewCmd.Flags().BoolVarP(&opts.Web, "web", "w", false, "Open mr in a browser. Uses default browser or browser specified in BROWSER variable")
commands/mr/issues/mr_issues.go:55:                     fmt.Fprintf(f.IO.StdOut, "%s\n%s\n", title.Describe(), issueutils.DisplayIssueList(f.IO, mrIssues, repo.FullName()))
docs/source/issue/create.rst:26:  $ glab issue new -t "Fix CVE-YYYY-XXXX" -l security --linked-mr 123
```

It appears that beside the auto-generated docs, there were no other places with similar changes to make. `issue_create.go:73,132` matches the 'test grep' because `--linked-mr`, so this is good, and similarly `mr_issues.go:55` appears intentional. So at least there is no other low hanging fruit

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation
- [ ] Chore (Related to CI or Packaging to platforms)

### CODE REVIEW

**Comment by justin-f-perez** [comment]
Oops... just noticed there's some documentation to update as well. will amend commit and push

File: docs/source/issue/view.rst [hunk 1]
```diff
@@ -32,11 +32,11 @@ Options
 
 ::
 
-  -c, --comments       Show mr comments and activities
+  -c, --comments       Show issue comments and activities
```
Comment (zemzale) [inline docs/source/issue/view.rst]: No need to change the docs here. They are generated automatically when creating a release using the `make gen-docs` command.

**Review by zemzale — CHANGES_REQUESTED** [review]
Thanks for the PR @justin-f-perez. One small thing about the docs, otherwise LGTM

File: docs/source/issue/view.rst (hunk 1 above)
Comment (justin-f-perez) [reply docs/source/issue/view.rst]: should be resolved now, thanks for noting it!

**Review by zemzale — APPROVED** [review]

**Pull request merged**